code/game/mecha/combat/combat.dm

```diff
@@ -17,6 +17,8 @@
 	max_special_equip = 1
 	cargo_capacity = 1
 
+	encumberence_gap = 1.5

```

Comment:
I hate to tell you, but the word is 'encumbrance'

Reply:
..Fuck.